### PR TITLE
Use @guardian/types for Display

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Replaces our local const enum version of `Display` with [the version](https://github.com/guardian/types/blob/main/src/format.ts#L17) exported from `@guardian/types/Format`

## Why?
This is part of a general move to use @guardian/types more so the definition of our content is better, and properly shared.

### CODE REVIEW

File: src/lib/content.d.ts [hunk 1]
```diff
@@ -33,10 +33,7 @@ interface BlockquoteBlockElement {
 
 interface CaptionBlockElement {
     _type: 'model.dotcomrendering.pageElements.CaptionBlockElement';
-    display: Display;
-    designType: DesignType;
     captionText?: string;
-    pillar: Pillar;
```
Comment (oliverlloyd) [inline src/lib/content.d.ts:39]: These were never used and should not have been here

File: src/lib/format.ts [hunk 2]
```diff
@@ -23,63 +17,3 @@ export const toTypesPillar = (p: Pillar): TypesPillar => {
             return TypesPillar.News;
     }
 };
-
-const toTypesDisplay = (d: Display): TypesDisplay => {
```
Comment (oliverlloyd) [inline src/lib/format.ts:27]: This function wasn't used and is no longer needed now that we import directly from /types

File: src/lib/format.ts [hunk 3]
```diff
@@ -23,63 +17,3 @@ export const toTypesPillar = (p: Pillar): TypesPillar => {
...
-        case Display.Immersive:
-            return TypesDisplay.Immersive;
-        case Display.Showcase:
-            return TypesDisplay.Showcase;
-        default:
-            return TypesDisplay.Standard;
-    }
-};
-
-const toTypesDesign = (d: DesignType): TypesDesign => {
```
Comment (oliverlloyd) [inline src/lib/format.ts:40]: This function wasn't used and is no longer needed now that we import directly from /types

File: src/lib/format.ts [hunk 4]
```diff
@@ -23,63 +17,3 @@ export const toTypesPillar = (p: Pillar): TypesPillar => {
...
-        case 'AdvertisementFeature':
-            return TypesDesign.AdvertisementFeature;
-
-        // note some are unsupported
-        default:
-            return TypesDesign.Article;
-    }
-};
-
-export const asFormat = (
```
Comment (oliverlloyd) [inline src/lib/format.ts:75]: This function wasn't used and is no longer needed now that we import directly from /types

File: src/web/components/AdSlot.tsx [hunk 5]
```diff
@@ -200,8 +200,9 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
                         </div>
                     );
                 }
+                default:
+                    return null;
```
Comment (oliverlloyd) [inline src/web/components/AdSlot.tsx:204]: Undefined is no longer an acceptable return type

**Review by liywjl — APPROVED** [review]
👍

**Comment by oliverlloyd** [comment]
[Use Display from @guardian/types](https://trello.com/c/vgPxdgyi/2264-use-display-from-guardian-types)

**Pull request merged**